js.go

```diff
@@ -1482,9 +1507,51 @@ func RateLimit(n uint64) SubOpt {
 }
 
 // BindStream binds a consumer to a stream explicitly based on a name.
-func BindStream(name string) SubOpt {
+func BindStream(stream string) SubOpt {
 	return subOptFn(func(opts *subOpts) error {
-		opts.stream = name
+		if opts.stream != "" {
+			return fmt.Errorf("nats: duplicate stream name (%s and %s)", opts.stream, stream)
+		}
+
+		opts.stream = stream
+		return nil
+	})
+}
+
+// BindOnly binds a subscription to a consumer from a stream without attempting to create.
+// The first argument is the stream name and the second argument will be the consumer name,
+// any extra arguments will be ignored.
+func BindOnly(params ...string) SubOpt {
```

Comment:
I think we make these explicit.

```go
Bind(stream, consumer string)
```

Reply:
when we do that, we will have to specify the durable name on pull consumer again.
With it being optional, we don't have to